[DOCS] 7.17.2 Release Notes (#1750)
elastic/security-docs: docs/release-notes/7.17.asciidoc

== 7.17

[discrete]
[[release-notes-7.17.2]]
=== 7.17.2

[discrete]
[[bug-fixes-7.17.2]]
==== Bug fixes and enhancements
* Fixes an {endpoint-sec} integration bug that prevented benign Windows files from being deleted under certain circumstances.

> * The primary signer for `elastic-endpoint.exe` has changed from ELASTICSEARCH B.V. to Elasticsearch, Inc.

Reviewer: Can this include a link to this issue? https://github.com/elastic/endpoint/issues/15

> * Ensures {endpoint-sec} can continue to run on all supported Windows versions by changing the primary signer of the `elastic-endpoint.exe` file from ELASTICSEARCH B.V. to Elasticsearch, Inc. (https://github.com/elastic/endpoint/issues/15[#15]).

Author: Yeah, I wasn't sure whether to offset the text or just keep it as is. I'm fine with the backticks. Don't want to risk someone assuming that the quotation marks are included in the signature.

> * Ensures {endpoint-sec} can continue to run on all supported Windows versions by changing the primary signer of the `elastic-endpoint.exe` file from `ELASTICSEARCH B.V.` to `Elasticsearch, Inc.` (https://github.com/elastic/endpoint/issues/15[#15]).

Reviewer 2: ```suggestion
* Ensures {endpoint-sec} continues to run on all supported Windows versions by changing the primary signer of the `elastic-endpoint.exe` file from `ELASTICSEARCH B.V.` to `Elasticsearch, Inc.` (https://github.com/elastic/endpoint/issues/15[#15]).
```

Author: I'll apply this across the board. Thanks, @benironside !

> * Backports the bug that prevented Endpoint Security 8.1.1 from being run on Windows 8.1 and Server 2012 R2. With this change, the primary signer for `elastic-endpoint.exe` has changed from ELASTICSEARCH B.V. to Elasticsearch, Inc. (https://github.com/elastic/endpoint/issues/15[#15]).

Reviewer: "backports the bug" sounds wrong 😄 

```suggestion
* Backports the fix for the bug that prevented Endpoint Security 8.1.1 from being run on Windows 8.1 and Server 2012 R2. With this change, the primary signer for `elastic-endpoint.exe` has changed from ELASTICSEARCH B.V. to Elasticsearch, Inc. (https://github.com/elastic/endpoint/issues/15[#15]).
```

Author: Changing to the following as discussed over Slack:
To ensure Endpoint Security continues to run on all supported Windows versions, the primary signer for `elastic-endpoint.exe` has changed from ELASTICSEARCH B.V. to Elasticsearch, Inc.

* Updates the minimum role permissions needed to import rules with actions. After this change, roles must have at least `Read` privileges for the `Actions and Connectors` feature to import rules with actions ({pull}126203[#126203]).

[discrete]
[[release-notes-7.17.1]]
=== 7.17.1

[discrete]
[[known-issue-7.17.1]]
==== Known issues

> * An {endpoint-sec} integration bug will prevent benign Windows files from being deleted under certain circumstances.

Reviewer 3: ```suggestion
* An {endpoint-sec} integration bug prevents benign Windows files from being deleted under certain circumstances.
```

== 7.17

[discrete]
[[release-notes-7.17.2]]
=== 7.17.2

[discrete]
[[bug-fixes-7.17.2]]
==== Bug fixes and enhancements
* Fixes an {endpoint-sec} integration bug that prevented benign Windows files from being deleted under certain circumstances.

> * The bug that prevented Endpoint Security from being run on Windows 8.1 and Server 2012 R2 has been fixed. With this change, the primary signer for `elastic-endpoint.exe` has changed from ELASTICSEARCH B.V. to Elasticsearch, Inc (https://github.com/elastic/endpoint/issues/15[#15]).

Reviewer: I'm sure it's typographically awful, but there is a period at the end of the new signer name

```suggestion
* The bug that prevented Endpoint Security from being run on Windows 8.1 and Server 2012 R2 has been fixed. With this change, the primary signer for `elastic-endpoint.exe` has changed from ELASTICSEARCH B.V. to Elasticsearch, Inc. (https://github.com/elastic/endpoint/issues/15[#15]).
```

Author: Good catch, thanks! I accidentally deleted the period when adding the PR link.

> * The bug that prevented Endpoint Security from being run on Windows 8.1 and Server 2012 R2 has been fixed. With this change, the primary signer for `elastic-endpoint.exe` has changed from ELASTICSEARCH B.V. to Elasticsearch, Inc (https://github.com/elastic/endpoint/issues/15[#15]).

Reviewer: Sorry for a 2nd edit here.

```suggestion
* The bug that prevented Endpoint Security 8.1.1 from being run on Windows 8.1 and Server 2012 R2 was backported. With this change, the primary signer for `elastic-endpoint.exe` has changed from ELASTICSEARCH B.V. to Elasticsearch, Inc (https://github.com/elastic/endpoint/issues/15[#15]).
```

* Updates the minimum role permissions needed to import rules with actions. After this change, roles must have at least `Read` privileges for the `Actions and Connectors` feature to import rules with actions ({pull}126203[#126203]).

[discrete]
[[release-notes-7.17.1]]
=== 7.17.1

[discrete]
[[known-issue-7.17.1]]
==== Known issues
* An {endpoint-sec} integration bug will prevent benign Windows files from being deleted under certain circumstances.

> * Endpoint Security cannot be run on Windows 8.1 and Server 2012 R2 (https://github.com/elastic/endpoint/issues/15[#15]).

Reviewer: Endpoint Security 7.17.1 works on all Windows variants. It's 8.1.1 that doesn't work on those two Windows versions.

```suggestion
```

Author: Removing since this isn't a bug with 7.17.1. It's being added to 7.17.2 as a preventative fix.
